pom.xml

```diff
@@ -93,7 +93,7 @@
         <protobuf.version>${protobuf-version}</protobuf.version>
         <retrofit.version>2.5.0</retrofit.version>
         <scala-2.11.version>2.11.12</scala-2.11.version><!-- Spark -->
-        <smallrye.reactive.messaging.camel.version>2.7.0</smallrye.reactive.messaging.camel.version>
+	<smallrye.reactive.messaging.camel.version>2.7.1</smallrye.reactive.messaging.camel.version> <!-- keep in sync with Quarkus SmallRye Reactive Messaging -->
```

Comment:
🕱 tab !?!?! 
```suggestion
        <smallrye.reactive.messaging.camel.version>2.7.1</smallrye.reactive.messaging.camel.version> <!-- keep in sync with Quarkus SmallRye Reactive Messaging -->
```

Reply:
Guess I deserved that for making a last moment edit using vim :stuck_out_tongue: